plugins/embed-optimizer/detect.js

```diff
@@ -51,19 +32,17 @@ const loadedElementContentRects = new Map();
  * @type {InitializeCallback}
  * @param {InitializeArgs} args Args.
  */
-export async function initialize( { isDebug } ) {
+export async function initialize( { logger } ) {
```

Comment:
See note above passing the logger functions directly without needing the `logger` object.